modernomad/settings.py

```diff
@@ -211,4 +212,21 @@
     '--nologcapture'
 ]
 
+
+class DisableMigrations(object):
+    def __contains__(self, item):
+        return True
+
+    def __getitem__(self, item):
+        return None
+
```

Comment:
I just copied and pasted this from a Stack Overflow answer, to be honest! Digging into it a bit further, it looks like this setting allows you to override the way that migrations are done for each application:

https://docs.djangoproject.com/en/1.11/ref/settings/#migration-modules

The default is an empty dictionary (i.e. do the default for everything). I'm guessing that setting the value to `None` means "don't run a migration". So, by overriding this with a fake dictionary that returns `None` for every key, it's saying "don't run a migration for any application".

django-test-without-migrations also takes the same approach: https://github.com/henriquebastos/django-test-without-migrations/blob/master/test_without_migrations/management/commands/_base.py

We could use that package, but seemed simpler just to use this little snippet, and also have a bit in our settings to customise test runs.